eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/SimpleEventStore.java

```diff
@@ -83,9 +80,14 @@ public EventStoreTransaction transaction(@Nonnull ProcessingContext processingCo
     public CompletableFuture<Void> publish(@Nullable ProcessingContext context,
                                            @Nonnull List<EventMessage<?>> events) {
         if (context == null) {
-            UnitOfWork unitOfWork = unitOfWorkFactory.create();
-            unitOfWork.runOnPostInvocation(c -> appendToTransaction(c, events));
-            return unitOfWork.execute();
+            AppendCondition none = AppendCondition.none();
+            List<TaggedEventMessage<?>> taggedEvents = new ArrayList<>();
+            for (EventMessage<?> event : events) {
+                taggedEvents.add(new GenericTaggedEventMessage<>(event, Set.of()));
```

Comment:
We don't apply the TagResolver here, because... otherwise it would influence models?

Reply:
I also don't understand it. Event if it's not a mistake, I think it should be explained here with some comment.

Reply:
I checked with the Server team, and it is indeed perfectly valid to append events with tags without validating them via an `AppendCondition`.
Hence, this PR should use the `TagResolver` at this point too. 
As we're currently rather busy, I anticipate to make this change somewhere next week.

Reply:
Added the tag resolver just now, so I'll resolve this comment!